Sources/UIComponent/Components/View/Image.swift

```diff
@@ -6,7 +6,7 @@ public struct Image: ViewComponent {
     public let image: UIImage
 
     public init(_ imageName: String) {
-        self.init(UIImage(named: imageName)!)
+        self.init(UIImage(named: imageName) ?? UIImage())
```

Comment:
Should we raise an assert here? So that it is easier to catch the issue in DEBUG